api/views.py

```diff
@@ -410,7 +411,8 @@ def handle_post(self, request, *args, **kwargs):
             return bad_request('Must include an `email` property')
 
         try:
-            user = User.objects.get(email=body['email'])
+            req_email = body['email'].lower()
+            user = User.objects.annotate(email_lower=Lower('email')).get(email_lower=req_email)
```

Comment:
[Same](https://github.com/IFRCGo/go-api/pull/754/files#r433796168) here.

Reply:
Thanks again! Totally forgot about that you mentioned this in another place too. Done: bed7976